core/src/test/java/io/questdb/test/griffin/AsOfJoinTest.java

```diff
@@ -798,6 +877,108 @@ public void testLtJoinSequenceGapOnKey() throws Exception {
         });
     }
 
+    @Test
+    public void testNestedASOF_keySymbol() throws Exception {
+        assertMemoryLeak(() -> {
+            compiler.setFullFatJoins(true);
+            compile("CREATE TABLE 'tests' (\n" +
+                    "  Ticker SYMBOL capacity 256 CACHE,\n" +
+                    "  ts timestamp\n" +
+                    ") timestamp (ts) PARTITION BY MONTH");
+            compile("insert into tests VALUES ('AAPL', '2000'),('AAPL', '2001'),('AAPL', '2002'),('AAPL', '2003'),('AAPL', '2004'),('AAPL', '2005')");
+            compile("insert into tests VALUES ('QSTDB', '2003'),('QSTDB', '2004'),('QSTDB', '2005'),('QSTDB', '2006'),('QSTDB', '2007'),('QSTDB', '2008')");
+
```

Comment:
It would be good  to format queries a bit .

Reply:
is the current version better?

Reply:
Yes 👍